inkscape: fix obsolete option, add -D example

SheldonZhong opened the pull request:
<!-- Thank you for sending a PR! -->
<!-- Please perform the following checks and mark all the boxes accordingly. -->
<!-- You can remove the checklist items that don't apply to your PR. -->

- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).


`-e` is obsolete option.

Added an example for output the drawing area `-D`, which should be very helpful to know.

`--verb` could receive more than one verbs.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tldr-pages/tldr?pullRequest=4279) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=4279) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=4279) it.</sub>

sbrl reviewed (approved):
Thanks, @SheldonZhong! I didn't even know that I new version of my favourite vector graphics editor had come out - awesome!

These edits look ok to me, but this is slightly awkward because the `-e` argument was changed. Is there a way we can work this into the description somehow (e.g. mentioning that it's Inkscape v1.0+)?

sbrl commented:
Note to whoever merges this from @waldyrious:

> Btw, whoever merges #4279, please include the description in the opening comment of the PR in the body of the squash-merge commit :)

Ref Btw, whoever merges #4279, please include the description in the opening comment of the PR in the body of the squash-merge commit :)

SheldonZhong commented:
> Thanks, @SheldonZhong! I didn't even know that I new version of my favourite vector graphics editor had come out - awesome!
> 
> These edits look ok to me, but this is slightly awkward because the `-e` argument was changed. Is there a way we can work this into the description somehow (e.g. mentioning that it's Inkscape v1.0+)?

- Added a line in the description to use `-e` for version 0.92.x.

- Removed the `-z` hints, ref https://wiki.inkscape.org/wiki/index.php/Using_the_Command_Line#Deprecations_and_Replacements


Though some of the repositories are still distributing 0.92.x, I think we should prefer Inkscape 1.0+ in TLDR page.

zdroid commented:
@SheldonZhong can you sign the CLA please?
